### ISSUE TITLE
[Elastic Agent]: Stand-alone Agent is not successfully enrolled. File beat does not run so UI status changes to Failed.

### ISSUE THREAD

**Issue by kamalpreetpahwa-qasource** [elastic/beats#19384]
**Kibana version:**
Kibana:8.0

**Elasticsearch version:**
Elasticsearch:8.0 

**Agent version:**
Agent:8.0

**Host OS and Browser version:**
Windows 10, Chrome

**Original install method (e.g. download page, yum, from source, etc.):**
From 8.0 snapshot

**Description**
[Ingest Management]: Agent is not enrolled successfully, as file beat application is not running and status changed to Failed.

**Preconditions**
1. Kibana 8.0 cloud environment should be available.

**Steps to Reproduce**
1. Open power shell and install the agent service with the ps1 script.
2. Then launch Kibana 8.0 environment link in browser and navigate to "Fleet" page.
3. Click "Enroll new agent" button and copy the string for default config.
4. Go back to power shell and then enroll the agent with enrollment string copied in step 4 having token.
5. Now start the agent.
6. Observe that error appears under Fleet tab next to host.
7. Click on host name and observe activity logs.
8. Also, "unknown error" appears under power shell window.

**Test data**
N/A

**Impacted Test case id**
N/A 

**Actual Result**
[Fleet]: Agent is not enrolled successfully, as file beat application is not running and status changed to Failed.

**Expected Result**
Agent should get enrolled successfully without any error.

**What's working**
N/A

**What's not working**
This issue is also occurring with 7.9 snapshot.

**Screenshot**
![8 0ErrorAgent](https://user-images.githubusercontent.com/61860613/85674720-b42e0380-b6e2-11ea-8104-e29d3b4cfa27.png)

**Logs**
![KIbana8 0Error](https://user-images.githubusercontent.com/61860613/85674830-cb6cf100-b6e2-11ea-969e-7d2e974a57fb.png)

**Comment by manishgupta-qasource** [comment]
Reviewed & assigned to @EricDavisX

**Comment by EricDavisX** [comment]
@ph @michalpristas this is a supported scenario, right?  starting from scratch and running the windows .ps1 install script then enrolling the agent?  I suppose it is only testable/ broken on Windows.

**Comment by michalpristas** [comment]
yes this is a valid configuration. 
can you please share:
- configuration which is send to agent (can be found in fleet in yml format)
- version of the agent (running `./elastic-agent version`)

**Comment by kamalpreetpahwa-qasource** [comment]
Hi @michalpristas 

We have re-verified the issue with today's latest 8.0 snapshot and observed the following.

Please find below the environment details and observations.

```
Kibana 8.0 snapshot

Elastic Agent 8.0.0-snapshot

HOST: Windows 10

```

Observation:
We observed that now agent is enrolled successfully with Windows 10 host on 8.0 snapshot. However, data is not get streamed after enrolling the agent.
Screenshot:
![8 0DatastreamIssue](https://user-images.githubusercontent.com/61860613/86108813-45d1b280-bae1-11ea-85a0-c102d21bd5ef.png)

Also, We found that "YAML" tab is missing on 8.0 cloud environment. [Please refer the attached screenshot]
![YAMLTAB8 0](https://user-images.githubusercontent.com/61860613/86108985-7ca7c880-bae1-11ea-8b4b-abd1e0e6323f.png)


Please look into the issue and let us know if any other information is required from our side.

Thanks,

**Comment by michalpristas** [comment]
are logs same as initial ones? containing any errors?

**Comment by kamalpreetpahwa-qasource** [comment]
Hi @michalpristas 

Today we verified the issue with 02 Windows 10 VMs and enrolled the agent 8.0 on Kibana 8.0 snapshot environment.
We observed that agent is enrolled successfully and also data is getting streamed as well under "Data Stream" tab.

Screenshot:
![19384_1stjuly](https://user-images.githubusercontent.com/61860613/86235685-6d467f00-bbb6-11ea-8d47-c1951089e38f.png)

Hence, we are closing this issue.

Thanks,
